content/articles/php-sessions-and-cookies-to-create-a-secure-remember-me-section-in-a-login-script/Index.md

```diff
@@ -0,0 +1,356 @@
+### Introduction
+In the Login Script, the 'Remember Me' functionality allows users who have logged in to keep their status. A user's logged-in status is serialized in PHP `sessions`, `cookies`, or other similar storage when they select the `Remember Me` option.
+In this edge, you will learn how to create a persistent PHP web application.
+#### Table of contents
+- [Introduction](#introduction)
+- [Preserve logged in state](#create-cookies-to-preserve-the-logged-in-state)
+- [validate remembered login](#validate-remembered-login-with-php-session-and-cookies)
+- [Clear remembered login](#clear-remembered-login-with-session-and-cookies-on-logout)
+- [Controller classes](#auth-and-dbcontroller-classes)
+- [Conclusion](#conclusion)
+- [check the code snippets at github](https://github.com/Juskam/secure-remember-me-for-login-using-php-session-and-cookies)
+Make sure to look at the source code on GitHub so you know what files to create.
+### Create cookies to preserve the logged-in state
+To obtain the `username` and `password`, I've created a login form. This form has a 'Remember Me' checkbox for the user's convenience, which allows the user to keep his logged-in state. When a user provides their login information, PHP receives it and compares it to the database of members.

```

Comment:
.....towards the last, "PHP receives it and compares it to database of members....." This sentence could be written in some other better ways. Remember we submit form to server which then processes that data???. database has records? why use `members`?